docs/keda-installation.md

```diff
@@ -1,8 +1,8 @@
-# Install Keda Manager
+# Install Keda Module
 
 ## Install Keda module manually
 
-1. To install Keda Manager manually, apply the following script:
+1. Apply the following script to install Keda Manager:
```

Comment:
Install Keda Manager or Keda Module (to align it with the headline)?

Same question applies to line 23 etc.


Maybe we could put the definition "Keda module is KEDA + Keda Manager" in one of the first lines of this doc?

Reply:
As far as I understand, when you install manually you need to install Keda Manager, and then it helps you install Keda.
I will add the definitions.